source/globalCommands.py

```diff
@@ -3304,12 +3307,38 @@ def script_braille_toggleTether(self, gesture):
 		# (braille can be tethered automatically or to either focus or review position).
 		ui.message(_("Braille tethered %s") % TetherTo(newTetherChoice).displayString)
 
+	@script(
+		# Translators: Input help mode message for toggle braille mode command
+		description=_('Toggles braille mode between follow cursors and display speech output'),

```

Comment:
I'd leave the options out of the descriptions. Such a thing would easily be overseen when adding a new mode.

Reply:
This is unresolved

```suggestion
		description=_("Toggles braille mode"),
```